packages/panzer/disc-fe/src/evaluators/Panzer_GatherSolution_Epetra_decl.hpp

```diff
@@ -126,7 +128,10 @@ class GatherSolution_Epetra<panzer::Traits::Residual,TRAITS,LO,GO>
   bool useTimeDerivativeSolutionVector_;
   std::string globalDataKey_; // what global data does this fill?
 
-  Teuchos::RCP<Epetra_Vector> x_;
+  Teuchos::RCP<Epetra_Vector>                   x_;
+  Teuchos::RCP<const Epetra_Map>                ownedMap_;
+  Teuchos::RCP<const Thyra::VectorBase<double>> xOwned_;
+  Teuchos::RCP<Epetra_Vector>                   xGhosted_;
```

Comment:
Instead of just `x_`, which contains all the owned and ghosted information, the new way will have `xOwned_` and `xGhosted_`, where the latter doesn't contain any duplicate information from the former.  We also need the `ownedMap_` to create an `Epetra_Vector` out of `xOwned_` when we need to.